corehq/apps/analytics/static/analytix/js/appcues.js

```diff
@@ -0,0 +1,49 @@
+/* global Appcues, Array, window */
+
+/**
+ * Instantiates the AppCues analytics and customer support messaging platform.
+ */
+hqDefine('analytix/js/appcues', [
+    'underscore',
+    'analytix/js/initial',
+    'analytix/js/logging',
+    'analytix/js/utils',
+], function (
+    _,
+    initialAnalytics,
+    logging,
+    utils
+) {
+    'use strict';
+    var _get = initialAnalytics.getFn('appcues'),
+        _logger = logging.getLoggerForApi('Appcues'),
+        _ready = $.Deferred();
+
+    $(function () {
+        var apiId = _get('apiId'),
+            scriptUrl = "//fast.appcues.com/" + apiId + '.js';
+
+        _ready = utils.initApi(_ready, apiId, scriptUrl, _logger, function () {
+            Appcues.identify(_get("userId"), {
```

Comment:
Users should be identified based on their email. The growth team doesn't work with couch user ids.